services/web/src/components/search/SearchProvider.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Loader, Message, Divider } from 'semantic';
 import { debounce, pickBy } from 'lodash';
-import Pagination from 'components/Pagination';
 
-export const SearchContext = React.createContext();
+import SearchContext from './context';
+import Pagination from './Pagination';
 
 export default class SearchProvider extends React.Component {
+  static Pagination = Pagination;
+
```

Comment:
why not also have static Filters  here too.
both the Pagination and Filters are UI

Reply:
i take it back thats gets ugly long <SearchProvider.Filters.Checkbox>

Reply:
yeah thought about that and came to the same conclusion